### DISCUSSION TITLE
New Supported Model: Havana (& feedback)

### DISCUSSION THREAD

**Discussion by Addonnis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#178] [General]
Confirming the Havana (6-wire) is supported, exactly as-per the build instructions. No further modifications (or 10K ohm resistor) needed.

Also just want to thank VisualApproach, I have donated. This was my first project involving a custom PCB/microcontroller/code and I'm grateful for the instructions to be made available for free.

Challenges I had personally:
- Navigating Visual Studio Code. Small learning curve if you've never used it before. I had issues installing the PlatformIO IDE extension which I finally resolved by clean-slating the VS Code application and all folders under %appdata%. Probably my fault. 
- The first USB cable I used for upload was not suitable. The instructions mention this ("power only" cables). Opening device manager and confirming the "USB serial" device is detected when I connected the ESP8266 helped.
- Finding the JST-SM connectors locally. I'm located in AU and found it really challenging to find someone stocking the 6-pin connectors in small volumes in AU at an affordable price. 
- Confirming the IP address of the unit after joining my wireless network, before connecting to the spa. The instructions said the IP would be listed in the 'serial monitor window' but I wasn't sure where to find this. I found by logging into my router and checking DHCP leases.

Thanks again VisualApproach - Looking forward to playing around with other projects on the net thanks to this experience.

**Comment by visualapproach** [comment]
Thank you @Addonnis !
Happy to hear you appreciated this! And thank you very much for donating and sharing your story 🙇‍♂️ !
The serial monitor is started by clicking this button:
![image](https://user-images.githubusercontent.com/22987250/149201429-1d8aea99-9bbc-4c39-8891-5506e78905ed.png)
